src/components/header/SkipToContent.js

```diff
@@ -1,8 +1,11 @@
-import { Link } from "react-router-dom"
+import { Link } from "react-router-dom";
+import { useRef } from 'react';
+
+const SkipToContent = ({ contentRef }) => {
+  const skipLink = useRef(null);
 
-const SkipToContent = () => {
   return (
-      <Link to="#content" className="skiplink">Skip to main content</Link>
+      <Link ref={skipLink} to="#content" onClick={() => { skipLink.current.blur(); contentRef.current.focus()}} className="skiplink">Skip to main content</Link>
   )
 };
```

Comment:
Let's do a quick check that skipLink and contentref current are not null here.

Reply:
A conditional statement has been added as requested above.